invenio_records_resources/services/files/tasks.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2022 CERN.
+#
+# Invenio-Records-Resources is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see LICENSE file for more
+# details.
+
+"""Files tasks."""
+
+import requests
+from celery import shared_task
+from invenio_access.permissions import system_identity
+
+from ...proxies import current_service_registry
+
+
+@shared_task(ignore_result=True)
+def fetch_file(service_id, record_id, file_key):
```

Comment:
minor: hate myself for missing this one, this should be added as an `invenio_celery.tasks` entrypoint in `setup.cfg`

Reply:
ooops done